src/interface/blas1/axpy.cpp.in

```diff
@@ -47,7 +47,20 @@ namespace internal {
 
 template typename SB_Handle::event_t _axpy(
     SB_Handle &sb_handle, ${INDEX_TYPE} _N, ${DATA_TYPE} _alpha,
-    ${container_t0} _vx, ${INCREMENT_TYPE} _incx, ${container_t1} _vy,
+    BufferIterator<${container_t0}> _vx, ${INCREMENT_TYPE} _incx, BufferIterator<${container_t1}> _vy,
     ${INCREMENT_TYPE} _incy, const typename SB_Handle::event_t& dependencies);
+
+#ifdef SB_ENABLE_USM
+template typename SB_Handle::event_t _axpy(
```

Comment:
The interface without 'const' simplifies the tests implementation and benchmarks too, as we don't have to add the `const` qualifier to all the input pointers (implicit cast does not work, I think because of how we use templates in `include/interface`).